book_source/04_appendix/06_devtools.Rmd

```diff
@@ -1,4 +1,4 @@
-### `devtools` package {#developer-devtools}
+# `devtools` package {#developer-devtools}
```

Comment:
Link to documentation https://devtools.r-lib.org/ instead of GitHub.com/hadley

Reply:
Addressed in 292fe96.